spec/features/visitor_can_browse_items_by_categories_spec.rb

```diff
@@ -3,7 +3,11 @@
 RSpec.describe "VisitorCanBrowseItemsByCategories", type: :feature do
   feature "visitor" do
     let!(:category) { Category.create(name: 'Lenses') }
-    let!(:item_one) { Item.create(name: '70-200 Canon L', description: "portrait lens", price: 12, category_id: category.id) }
+    let!(:item_one) { Item.create(name: '70-200 Canon L',
+                                  description: "portrait lens",
+                                  price: 12,
+                                  category_id: category.id,
+                                  image_url: 'http://www.bhphotovideo.com/images/images1000x1000/sony_sel1635z_vario_tessar_t_e_16_35mm_1082052.jpg') }
```

Comment:
Line is too long. [151/80]<br>Expression at 10, 151 should be on its own line.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.